pubspec.lock

```diff
@@ -186,6 +221,48 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "4.0.0"
+  image_picker:
```

Comment:
Why some many different package of image picker is used? Don't they provide same functionality.